webroot/src/components/CollapseCaretButton/CollapseCaretButton.less

```diff
@@ -0,0 +1,9 @@
+//
```

Comment:
_Testing find - just commenting here for a resolvable comment_

`yarn build` gives new warnings: ` WARNING  Compiled with 5 warnings`

It looks like those CSS build warnings that have trouble resolving multiple import orders. At a glance, it looks like the `ShowPasswordEye` & `HidePasswordEye` have been imported in a new order relative to `InputCheckbox`.

I suspect in `FinalizePrivilegePurchase.ts` that moving the import of `InputCreditCard` to after `InputCheckbox` may do the trick.